server/controllers.js

```diff
@@ -45,20 +40,25 @@ exports.updateMenu = (req, res) => {
           link: req.body.link,
         });
       }
+      console.timeEnd('updateMenuTimeLength');
     })
     .catch((err) => {
       res.status(400);
       res.send(err);
+      console.timeEnd('updateMenuTimeLength');
     });
 };
 
 exports.deleteMenu = (req, res) => {
+  console.time('deleteMenuTimeLength');
   menus.destroy({ where: { id: req.params.id } })
     .then(() => {
       res.send('Menu deleted');
+      console.timeEnd('deleteMenuTimeLength');
     })
     .catch((err) => {
       res.status(400);
       res.send(err);
+      console.timeEnd('deleteMenuTimeLength');
```

Comment:
all the console.time/console.timeEnd functions may be useful for development, but either comment out or remove them if they're not necessary for production